src/main/java/frc/robot/subsystems/conveyor/commands/ConveyorDefaultCommand.java

```diff
@@ -0,0 +1,38 @@
+package frc.robot.subsystems.conveyor.commands;
+
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Ports;
+import frc.robot.subsystems.example.Deez_Nuts;
+
+public class ConveyorDefaultCommand extends CommandBase {
+    private final Deez_Nuts ConveyorDefaultCommand;
+    private final WPI_TalonSRX motor = new WPI_TalonSRX(Ports.Conveyor.AUX);
+//    private frc.robot.subsystems.conveyor.Conveyor.AllianceColor color;
+
+    public ConveyorDefaultCommand(Deez_Nuts ConveyorDefaultCommand) {
```

Comment:
Why do you include "default" and "command" here? A command's name should describe what it does, this name doesn't do that.